hydrosdk/servable.py

```diff
@@ -116,6 +132,19 @@ def __init__(self, cluster, model, servable_name, metadata=None):
         self.meta = metadata
         self.cluster = cluster
 
+    def status(self):
```

Comment:
I think it'll be better to rename the method to `retrieve_status` and add a `self.status` field to `Servable.__init__`. This way we differentiate the action of retrieving updated status of a servable, with a status which was defined at the backend lookup.

Moreover, the implementation duplicates method `Servable.get`, but provides only a small portion of fetched data, for instance, servable `status` message from the backend also contains description message, which can be proven useful e.g. for debug.

On the whole, I'd suggest adding `status` and `status_message` fields to `Servable` and using `Servable.get` in order to poll new info about servable.

Reply:
👌🏻